app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> None:
+
+    masks_to_buy = 0
+
+    for person in friends:
+        try:
+            cafe.visit_cafe(person)
+
+        except VaccineError:
+            return "All friends should be vaccinated"
+
+        except NotWearingMaskError:
+            masks_to_buy += 1
+
+    if masks_to_buy > 0:
```

Comment:
You can simplify to " if masks_to_buy: " because if the value is zero it will automatically be False